mesheryctl/internal/cli/root/pattern/apply.go

```diff
@@ -222,13 +224,26 @@ mesheryctl pattern apply [pattern-name]
 			return err
 		}
 
+		type pattern struct {
+			Name     string                    `yaml:"name"`
+			Services unstructured.Unstructured `yaml:"services"`
+		}
+
+		var p pattern
+		err = yaml.Unmarshal([]byte(patternFile), &p)
+		if err != nil {
+			return errors.Wrap(err, "could not unmarshal pattern file")
+		}
+
+		s := utils.NewLoader("Applying pattern " + p.Name)
```

Comment:
Why not simply use `path.Base(file)` to display name here? We also use it to save the pattern file. 

https://github.com/meshery/meshery/blob/f4fbefd8708d8b535e6bcf6cd80c3d9dbb458187/mesheryctl/internal/cli/root/pattern/apply.go#L114